tap_facebook/__init__.py

```diff
@@ -210,16 +250,20 @@ def do_request():
 
 def get_start(state, tap_stream_id, bookmark_key):
     current_bookmark = singer.get_bookmark(state, tap_stream_id, bookmark_key)
-    LOGGER.info("found current bookmark %s", current_bookmark)
     if current_bookmark is None:
-        LOGGER.info("using start_date instead...%s", CONFIG['start_date'])
-        return CONFIG['start_date']
-    return current_bookmark
+        if tap_stream_id in INCREMENTAL_STREAMS:
+            LOGGER.info("no bookmark found for %s, will perform full sync...", tap_stream_id)
+            return pendulum.min
```

Comment:
Is there any chance this datetime value could make its way into the state? I don't want such an implementation specific detail as the minimum date to make its way into the state, because that could present a migration issue if we decide to use something other than pendulum. Could we instead keep the start as None and change the code that does the comparison so that it does a full sync when it sees None?

Reply:
That should be pretty doable since I pulled the datetime parsing into here.